tests/test_elements/test_mol2any/test_mol2maccs_key_fingerprint.py

```diff
@@ -0,0 +1,134 @@
+"""Tests for the MolToMACCSFP pipeline element."""
+
+from __future__ import annotations
+
+import unittest
+from typing import Any
+
+import numpy as np
+
+from molpipeline import Pipeline
+from molpipeline.any2mol import SmilesToMol
+from molpipeline.mol2any import MolToMACCSFP
+
+# pylint: disable=duplicate-code
+# Similar to test_mol2morgan_fingerprint.py and test_mol2path_fingerprint.py
+
+test_smiles = [
+    "c1ccccc1",
+    "c1ccccc1C",
+    "NCCOCCCC(=O)O",
+]
+
+
+class TestMolToMACCSFP(unittest.TestCase):
+    """Unittest for MolToMACCSFP, which calculates MACCS Key Fingerprints."""
+
+    def test_can_be_constructed(self) -> None:
+        """Test if the MolToMACCSFP pipeline element can be constructed.
+
+        Returns
+        -------
+        None
+        """
+        mol_fp = MolToMACCSFP()
+        mol_fp_copy = mol_fp.copy()
+        self.assertTrue(mol_fp_copy is not mol_fp)
+        for key, value in mol_fp.get_params().items():
+            self.assertEqual(value, mol_fp_copy.get_params()[key])
+        mol_fp_recreated = MolToMACCSFP(**mol_fp.get_params())
+        for key, value in mol_fp.get_params().items():
+            self.assertEqual(value, mol_fp_recreated.get_params()[key])
+
+    def test_sparse_dense_accordance(self) -> None:
```

Comment:
this test is redundant with the next one called `test_output_types`

Reply:
Done